lib/JSON/Unmarshal.rakumod

```diff
@@ -88,7 +88,7 @@ Please see the LICENCE file in the distribution
 
 =end pod
 
-our class X::CannotUnmarshal is Exception is export {
+our class X::CannotUnmarshal is Exception {
```

Comment:
You don't expect the client to consume or generate this class, right?

Reply:
With `is export` you only get 'CannotUnmarshal` exported, not `X::CannotUnmarshal`. Therefore it makes more sense to access it with full name `JSON::Unmarshal::X::CannotUnmarshal`.